docs/data/material/discover-more/showcase/showcase.md

```diff
@@ -1,13 +1,13 @@
 # Showcase
 
-<p class="description">The following is a list of some of the public apps using MUI.</p>
+<p class="description">Check out these public apps using Material UI to get inspired for your next project.</p>
 
-Are you using MUI? [Share your product](https://github.com/mui/material-ui/issues/22426).
+Are you working with MUI's component libraries? [Show us what you're building](https://github.com/mui/material-ui/issues/22426)! We'd love to see it.
 
-The websites we include rank well in at least one of these dimensions:
+To be featured on this page, your app should meet at least one of the following requirements:
```

Comment:
Should we emphasize that we don't look for PRs on this page? One attempt:

```suggestion
The apps listed on this page have been curated by MUI's maintainers. Each app should meet at least one of the following requirements:
```

Reply:
Ah ok, that's a good idea. Is there some other way for people to submit their app for consideration? Or are we trying to discourage that?

Reply:
Then again, if we don't want people to submit their apps here, then maybe we don't need to list any criteria at all? We can just say something like "This is a curated list of some of the best apps we've found that show off what's possible with Material UI."